test/general/build_targets/package_test.dart

```diff
@@ -99,11 +99,23 @@ void main() {
 
       processManager.addCommand(FakeCommand(
         command: <String>[
-          'dotnet',
-          'build',
-          '-c',
+          '/tizen-studio/tools/tizen-core/tz',
+          'set',
+          '-b',
           'Release',
-          '/p:DefineConstants=COMMON_PROFILE',
+          '-s',
+          'test_profile',
+          '-w',
+          '${projectDir.path}/tizen',
+        ],
+        onRun: (_) {},
+      ));
+
+      processManager.addCommand(FakeCommand(
```

Comment:
You could add multiple commands at once (`FakeProcessManager.addCommands()`).

Reply:
done!